### PR TITLE
Explain atlas visualisation script slightly more

### PR DESCRIPTION
**Pull request by adamltyson**
It wasn't 100% clear what the original text meant.

### CODE REVIEW

File: docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ inspect_meshes_folder("~/.brainglobe/temp/allen_mouse_10um_v1.0/meshes")
 Once an atlas is created with BrainGlobe’s atlas generation tools, it can be used with most software from the BrainGlobe software suite.
 These include [brainrender](/documentation/brainrender/index) and [brainrender-napari](/tutorials/visualise-atlas-napari) which provide a convenient way to visually inspect the generated atlas meshes.
 
-For example, you can visualise various parts of the Max Planck Zebrafish Brain Atlas next to each other by running the script below. You can do the same with your own newly-packaged atlas (or any other BrainGlobe atlas) by replacing as explained in the comments.
+For example, you can visualise various parts of the Max Planck Zebrafish Brain Atlas next to each other by running the script below. You can do the same with your own newly-packaged atlas (or any other BrainGlobe atlas) by replacing the atlas name as explained in the comments below:
```
Comment (alessandrofelder) [inline docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md:155]:
```suggestion
For example, you can visualise various parts of the Max Planck Zebrafish Brain Atlas next to each other by running the script below. You can do the same with your own newly-packaged atlas (or any other BrainGlobe atlas) by replacing the atlas name, the name of the additional reference, and the acronym of the structure you're interested in, as explained in the comments below:
```
More accurate but more wordy... not sure?

Comment (alessandrofelder) [reply docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md:155]: Also seems I forgot to replace "<structure_id>" with "root" in the python snippet :facepalm:

**Review by alessandrofelder — APPROVED** [review]
Now it's not technically clear there's more than the atlas name that needs replacing :thinking: not sure what the best formulation is, but I made a wordy suggestion.

File: docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md (hunk 1 above)
Comment (adamltyson) [reply docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md]: I'd vote for "wordy, but correct" over "vague" any day!/

Comment (adamltyson) [reply docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md]: Can I leave it with you to fix the snippet?

**Pull request merged**